backend-api/src/functions/asyncIssueBiometricCredential/asyncIssueBiometricCredentialHandler.test.ts

```diff
@@ -1049,94 +1052,339 @@ describe("Async Issue Biometric Credential", () => {
+              ipAddress: undefined,
+              redirect_uri: undefined,
+              sub: mockSubjectIdentifier,
+              suspected_fraud_signal: undefined,
+              txmaAuditEncoded: undefined,
+              flaggedRecord: undefined,
+              flags: undefined,
+              evidence: [
+                {
+                  type: "IdentityCheck",
+                  txn: "mockTxn",
+                  strengthScore: 0,
+                  validityScore: 0,
+                  activityHistoryScore: 0,
+                  checkDetails: [
+                    {
+                      checkMethod: "bvr",
+                      identityCheckPolicy: "published",
+                      activityFrom: undefined,
+                      biometricVerificationProcessLevel: 0,
+                    },
+                  ],
+                  txmaContraIndicators: [],
+                },
+              ],
+              credentialSubject: mockCredentialSubject,
+            },
+          );
+        });
+
+        it("Writes DCMAW_ASYNC_CRI_END event to TxMA", () => {
+          expect(mockWriteGenericEventSuccessResult).toHaveBeenNthCalledWith(
+            2,
+            {
+              eventName: "DCMAW_ASYNC_CRI_END",
+              componentId: mockIssuer,
+              getNowInMilliseconds: Date.now,
+              sessionId: mockSessionId,
+              govukSigninJourneyId: mockGovukSigninJourneyId,
+              ipAddress: undefined,
+              redirect_uri: undefined,
+              sub: mockSubjectIdentifier,
+              suspected_fraud_signal: undefined,
+              txmaAuditEncoded: undefined,
+            },
+          );
+        });
+
+        it("Logs COMPLETED with persistent identifiers", () => {
+          expect(consoleInfoSpy).toHaveBeenCalledWithLogFields({
+            messageCode: "MOBILE_ASYNC_ISSUE_BIOMETRIC_CREDENTIAL_COMPLETED",
+            persistentIdentifiers: {
+              sessionId: mockSessionId,
+              biometricSessionId: mockBiometricSessionId,
+              govukSigninJourneyId: mockGovukSigninJourneyId,
+            },
+          });
+        });
       });
 
-      it("Passes correct arguments to sendMessageToSqs (verifiable credential)", async () => {
-        expect(mockSendMessageToSqsSuccess).toHaveBeenCalledWith(
-          "mockIpvcoreOutboundSqs",
-          {
-            "https://vocab.account.gov.uk/v1/credentialJWT": [mockSignedJwt],
-            state: mockClientState,
-            sub: mockSubjectIdentifier,
+      describe("Given generated biometric credential has flags and flaggedRecord", () => {
+        const mockAuditDataWithFlags = {
+          contraIndicatorReasons: [],
+          txmaContraIndicators: [],
+          flags: ["FLAG_1", "FLAG_2"],
+          flaggedRecord: {
+            recordId: "mock-record-id",
+            reason: "mock-reason",
+            timestamp: "2024-01-01T12:00:00Z",
```

Comment:
Nitpick that got lost in the PR updates - this will make it more obvious that this is intended to be a placeholder object rather than a valid flagged record
```suggestion
          flaggedRecord: {
            mockKey: "mockValue"
```